src/noisepy/seis/S1_fft_cc_MPI.py

```diff
@@ -60,6 +61,12 @@ def cross_correlate(
         raw_store: Store to load data from
         fft_params: Parameters for the FFT calculations
         cc_store: Store for saving cross correlations
+        scheduler: Scheduler to use for parallelization
+        pair_filter: Function to decide whether a pair of channels should be used or not. E.g.
```

Comment:
@mdenolle @niyiyu @kuanfufeng , I wanted to include a simple example. Does this one seem appropriate to you?

Reply:
Looks good to me!

Reply:
this looks like the good example we talked about yesterday!